resources/public/api/conf/2.0/application.raml

```diff
@@ -0,0 +1,64 @@
+#%RAML 2.0
```

Comment:
@edjannoo 
Could you please check if you see anything wrong in this RAML definition?

Reply:
See comments about using a hierarchical structure in the RAML.
Also I see that you only have `GET` resources in here - one of the issues that we had with WSO2 was with resources having the following patterns:
```
POST /employment
GET /employment/{employmentId}
PUT /employment/{employmentId}
```
Make sure that you cover this type of pattern in your testing of the WSO2 patches

Reply:
I agreed with Ed we do not need extra endpoints.
In fact, if the APIs are designed without conflicting endpoints (WSO2 limitation), the APIs will work as expected.